modules/hifiasm/functions.nf

```diff
@@ -0,0 +1,70 @@
+/*
+ * -----------------------------------------------------
```

Comment:
Old version of `functions.nf`. Will need to re-install the latest version of tools.

Reply:
Yup, fixed!